Add Makefile

uqichi opened the pull request:
# issues
- [x] [makeコマンド一発でセットアップ](https://github.com/chkkchy/dotfiles/issues/4)
- [x] [READMEをジョブに切り出す](https://github.com/chkkchy/dotfiles/issues/3)
- [x] [mas導入](https://github.com/chkkchy/dotfiles/issues/1)

# todo
- [ ] `:GoInstallBinaries`実行をvimrcに書きたい。proの方と比較しよう
- [x] zprezto系のrcファイルのsymlinkの方法改善
- [x] ./dotfiles/.zshrcとかちゃんとsymlinkできてない。そもそもこれあったほうがいいのか？...
- [ ] Mac OSの設定。キーリピートの速さとかdefaults ...

# links
- [b4b4r07/dotfiles: dotfiles ❤ Testing my dotfiles repo on OS X to get my work environment ready in just a few moments\. \#VIM \+ \#ZSH \+ \#TMUX = Best Developer Environment](https://github.com/b4b4r07/dotfiles)
- [brewfileとかmasとか](http://qiita.com/takeo-asai/items/29724f94e2992fdc7246)
- [ ] [iTerm2の設定を複数のMacで共有する \- 日々精進](http://anton0825.hatenablog.com/entry/20140117/1390148165)

Commit:
Add Makefile

Commit:
[from now] 2016/12/01 15:17:11

 Makefile     | 13 +++++++++++--
 _brew.sh     | 56 +++++++++++++++++++++++++++++++++++++++++++++++++++-----
 _brewcask.sh | 22 ++++++++++++++++++----
 _setup.sh    | 19 -------------------
 4 files changed, 80 insertions(+), 30 deletions(-)

Commit:
[from now] 2016/12/01 21:17:05

 Makefile            |  7 +++---
 _brew.sh            | 61 -----------------------------------------------------
 _brewcask.sh        | 27 ------------------------
 _lnvim.sh           | 14 ------------
 scripts/brew.sh     | 61 +++++++++++++++++++++++++++++++++++++++++++++++++++++
 scripts/brewcask.sh | 27 ++++++++++++++++++++++++
 scripts/lnvim.sh    | 14 ++++++++++++
 7 files changed, 106 insertions(+), 105 deletions(-)

Commit:
[from now] 2016/12/01 21:24:51

 Makefile | 10 ++++++++++
 1 file changed, 10 insertions(+)

Commit:
[from now] 2016/12/01 21:52:34

 .gitmodules |  3 +++
 Makefile    |  3 +++
 README.md   | 23 ++---------------------
 zpresto     |  1 +
 4 files changed, 9 insertions(+), 21 deletions(-)

Commit:
[from now] 2016/12/02 10:54:28

 .gitmodules | 4 ++--
 Makefile    | 1 +
 zpresto     | 1 -
 zprezto     | 1 +
 4 files changed, 4 insertions(+), 3 deletions(-)

Commit:
[from now] 2016/12/02 11:03:00

 .zlogin    | 1 +
 .zlogout   | 1 +
 .zpreztorc | 1 +
 .zprofile  | 1 +
 .zshenv    | 1 +
 .zshrc     | 1 +
 6 files changed, 6 insertions(+)

Commit:
[from now] 2016/12/02 11:11:26

 .vim/template/template.html | 22 ++++++++++++++++++++++
 .vim/template/template.sh   |  1 +
 .vimrc                      |  6 +++---
 Makefile                    |  1 +
 scripts/lnvim.sh            | 14 --------------
 vim/template.go             |  7 -------
 vim/template.html           | 22 ----------------------
 vim/template.sh             |  1 -
 8 files changed, 27 insertions(+), 47 deletions(-)

Commit:
[from now] 2016/12/02 14:26:48

 .tmuxinator/tmuxinator.zsh                         |   29 +
 Makefile                                           |   28 +-
 dash/library.dash                                  |  Bin 45056 -> 0 bytes
 etc/dash/library.dash                              |  Bin 0 -> 45056 bytes
 etc/iterm2/com.googlecode.iterm2.plist             | 5999 ++++++++++++++++++++
 etc/iterm2/solarized/README.md                     |   63 +
 etc/iterm2/solarized/Solarized Dark.itermcolors    |  213 +
 etc/iterm2/solarized/Solarized Light.itermcolors   |  213 +
 ...om-solarized-dark-for-prezto-syntax.itermcolors |  213 +
 ...2b5-761c-49fb-a2e7-07405a271d61-2016-11-14.opml |  110 +
 etc/scripts/brew.sh                                |   61 +
 etc/scripts/brewcask.sh                            |   27 +
 iterm2/com.googlecode.iterm2.plist                 | 5999 --------------------
 iterm2/solarized/README.md                         |   63 -
 iterm2/solarized/Solarized Dark.itermcolors        |  213 -
 iterm2/solarized/Solarized Light.itermcolors       |  213 -
 ...om-solarized-dark-for-prezto-syntax.itermcolors |  213 -
 ...2b5-761c-49fb-a2e7-07405a271d61-2016-11-14.opml |  110 -
 scripts/brew.sh                                    |   61 -
 scripts/brewcask.sh                                |   27 -
 tmuxinator/tmuxinator.zsh                          |   29 -
 21 files changed, 6941 insertions(+), 6943 deletions(-)

Commit:
[from now] 2016/12/02 16:54:23

 etc/iterm2/com.googlecode.iterm2.plist | 95 +++++++++++-----------------------
 1 file changed, 31 insertions(+), 64 deletions(-)

Commit:
[from now] 2016/12/02 17:26:48

 etc/iterm2/com.googlecode.iterm2.plist | 59 +++++++++++++++++++++++++++++++---
 1 file changed, 55 insertions(+), 4 deletions(-)

Commit:
Add bin

uqichi committed:
[from now] 2016/12/02 19:28:26

 .gitconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

uqichi committed:
[from now] 2016/12/02 19:46:17

 doc/gradient1.png                                  | Bin 0 -> 175076 bytes
 etc/iterm2/itermcolors/Solarized Dark.itermcolors  | 213 +++++++++++++++++++++
 etc/iterm2/itermcolors/Solarized Light.itermcolors | 213 +++++++++++++++++++++
 etc/iterm2/profiles_window_bgimage.png             | Bin 0 -> 175076 bytes
 etc/iterm2/solarized/README.md                     |  63 ------
 etc/iterm2/solarized/Solarized Dark.itermcolors    | 213 ---------------------
 etc/iterm2/solarized/Solarized Light.itermcolors   | 213 ---------------------
 ...om-solarized-dark-for-prezto-syntax.itermcolors | 213 ---------------------
 8 files changed, 426 insertions(+), 702 deletions(-)

uqichi committed:
[from now] 2016/12/02 19:46:43

 etc/iterm2/com.googlecode.iterm2.plist | 16 +++++++---------
 1 file changed, 7 insertions(+), 9 deletions(-)

uqichi committed:
[from now] 2016/12/02 19:48:08

 doc/gradient1.png | Bin 175076 -> 0 bytes
 1 file changed, 0 insertions(+), 0 deletions(-)

uqichi committed:
Refac makefile

uqichi committed:
Modify submodule

uqichi committed:
[from now] 2016/12/03 02:26:28

 Makefile                               |  17 +-
 etc/iterm2/com.googlecode.iterm2.plist | 420 ++++++++++++++++-----------------
 2 files changed, 223 insertions(+), 214 deletions(-)

uqichi committed:
Add Brewfile

uqichi committed:
Fix

uqichi committed:
[from now] 2016/12/03 04:08:02

 .vim/template/template.html    | 22 ----------------------
 .vim/template/template.sh      |  1 -
 Makefile                       | 13 +++----------
 etc/vim/template/template.html | 22 ++++++++++++++++++++++
 etc/vim/template/template.sh   |  1 +
 5 files changed, 26 insertions(+), 33 deletions(-)

uqichi committed:
Modify help cmd

uqichi committed:
[from now] 2016/12/03 04:28:29

 Makefile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

uqichi committed:
[from now] 2016/12/03 04:30:47

 bin/README.md | 21 ---------------------
 1 file changed, 21 deletions(-)

uqichi committed:
[from now] 2016/12/03 12:04:28

 Makefile            |  6 +++---
 etc/scripts/dein.sh | 14 --------------
 etc/scripts/nvim.sh | 14 ++++++++++++++
 3 files changed, 17 insertions(+), 17 deletions(-)

uqichi committed:
[from now] 2016/12/03 12:24:36

 com.googlecode.iterm2.plist | Bin 0 -> 34519 bytes
 1 file changed, 0 insertions(+), 0 deletions(-)

uqichi committed:
[from now] 2016/12/03 12:40:06

 README.md                   |  10 ++++++----
 com.googlecode.iterm2.plist | Bin 34519 -> 0 bytes
 2 files changed, 6 insertions(+), 4 deletions(-)

chkkchy commented:
https://github.com/simonwhitaker/gitignore-boilerplates

chkkchy commented:
http://dotfiles.github.io/

chkkchy commented:
Backup, restore, and sync the prefs and settings for your toolbox. Your dotfiles might be the most important files on your machine.

chkkchy commented:
Mac OSの設定。
キーリピートの速さとか
defaults ...

chkkchy commented:
brew doctor

chkkchy commented:
$ xcode-select --install
$ sudo xcodebuild -license
※ `license agreements` が表示されるので `agree` を入力してEnterを押す

chkkchy commented:
http://yuukiar.co/blog/2015/01/08/setup-mac-os-x-yosemite/

uqichi committed:
[from now] 2016/12/04 13:00:05

 Makefile               | 6 +++++-
 README.md              | 2 +-
 etc/prezto/install.zsh | 4 ++++
 3 files changed, 10 insertions(+), 2 deletions(-)

uqichi committed:
[from now] 2016/12/04 20:53:06

 .zlogin                | 2 +-
 .zlogout               | 2 +-
 .zpreztorc             | 2 +-
 .zprofile              | 2 +-
 .zshenv                | 2 +-
 .zshrc                 | 2 +-
 .zshrc.me              | 2 ++
 Makefile               | 2 --
 etc/prezto/install.zsh | 4 ----
 9 files changed, 8 insertions(+), 12 deletions(-)

uqichi committed:
[from now] 2016/12/04 21:06:39

 etc/iterm2/colors/Default.itermcolors              | 344 +++++++++++++++++++++
 etc/iterm2/colors/Hybrid.itermcolors               | 213 +++++++++++++
 etc/iterm2/colors/Solarized Dark.itermcolors       | 213 +++++++++++++
 etc/iterm2/colors/Solarized Light.itermcolors      | 213 +++++++++++++
 etc/iterm2/itermcolors/Solarized Dark.itermcolors  | 213 -------------
 etc/iterm2/itermcolors/Solarized Light.itermcolors | 213 -------------
 6 files changed, 983 insertions(+), 426 deletions(-)

uqichi committed:
[from now] 2016/12/04 21:58:53

 Makefile               | 34 +++++++++++-----------------------
 etc/dash/init.sh       | 10 ++++++++++
 etc/nvim/init.sh       | 14 ++++++++++++++
 etc/scripts/nvim.sh    | 14 --------------
 etc/tmuxinator/init.sh |  3 +++
 5 files changed, 38 insertions(+), 37 deletions(-)

uqichi committed:
[from now] 2016/12/04 22:02:34

 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

chkkchy commented:
- :GoInstallBinaries実行をvimrcに書きたい。
- Mac OSの設定。キーリピートの速さとかdefaults ...
-  iTerm2の設定を複数のMacで共有する

上記は別PRで対応する

uqichi committed:
[from now] 2016/12/05 10:49:06

 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

uqichi committed:
Add iterm2 ss
